visualization/app/codeCharta/ui/metricChooser/metricChooser.component.scss

```diff
@@ -79,26 +79,35 @@ edge-chooser-component {
 		display: initial;
 	}
 
+	.search-bar {
+		background-color: #f5f5f5;
+		padding: 10px;
+		border: none rgb(0, 150, 136);
+		border-bottom: 2px solid;
+		color: rgb(0, 150, 136);
+	}
+
+	.search-icon {
+		position: absolute;
+		left: 0;
+		width: 12%;
+		height: $optionHeight - 8 + px;
+		top: 8px;
+		color: rgb(0, 150, 136);
+		box-shadow: 0 1px 5px 0 rgba(0, 0, 0, 0.2) !important;
+	}
+
 	.metric-search {
 		display: block;
 		position: fixed;
 		top: 8px;
-		width: 100%;
-		padding: 10px;
+		left: 12%;
+		width: 90%;
 		height: $optionHeight - 8 + px;
 		outline: none;
-		border: none;
-		border-bottom: solid 2px;
-		border-color: rgba(0, 0, 0, 0.12);
-		background-color: #f5f5f5;
 		font-family: Arial, FontAwesome, sans-serif;
 		font-size: 11pt;
-		box-shadow: 0 1px 5px 0 rgba(0, 0, 0, 0.2) !important;
-
-		&:focus {
```

Comment:
Was it intentionally to remove the focus effect?

Reply:
yes because the icon was not reacting with the search input bar so to keep the interaction clean. i have removed this.

Reply:
Sorry, I am not certain I understand this properly. The focus part should only be triggered in case the element is focused. It should not have any other implications?

Reply:
the focus style works only for the search input however not for the search icon.
![image](https://user-images.githubusercontent.com/57844849/144828355-803243e2-2c1e-4f7f-92e1-6fbd73644ce0.png)

Reply:
It is possible to manipulate the outer `div` with `focus:within`.